### DISCUSSION TITLE
"Docker engine is not running" error with running Sedge on Ubuntu 20.04 and Docker Desktop 4.25.1

### DISCUSSION THREAD

**Discussion by jan-o-e** [NethermindEth/sedge#335] [Q&A] [answered]
Currently having issues running Sedge on Ubuntu 20.04 with an Intel NUC (Intel i5). Latest version of Docker desktop is installed and upon following the docs and installing sedge, we obtain the following errors:

```bash
sedge deps install
[INFO] All dependencies are installed
```

```bash 
sedge deps check
[INFO] docker is installed
[WARN] Docker engine is not running. Please start it and try again
```

This is strange insofar as Docker Desktop and the Docker Engine are clearly running and I can run docker hello-world successfully. The same error (docker daemon not running) occurs later when I try and start sedge after configuring my validator settings.

Anyone have any suggestions for why this is?

Thanks.

**Comment by cbermudez97** [comment]
Hi @jan-o-e. Can you check that docker is accesible for the user running Sedge? If you need to use `sudo` for docker commands you will probably also need to run `sedge` commands with it.

**Reply by jan-o-e** [reply, marked as answer]
Was a docker issue unrelated to Sedge, thanks for the reply though. This fixed it for me:

```bash
sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plugin
